test/test.rs

```diff
@@ -58,7 +58,8 @@ fn read_exact<Fd: AsFd>(f: Fd, buf: &mut [u8]) {
 }
 
 /// Any test that creates child processes must grab this mutex, regardless
```

Comment:
Maybe we should update this comment to something like:

```suggestion
/// Any test that creates child processes or can be affected by child processes must grab this mutex, regardless
```